examples/regression/2-advanced-analysis/plot_timeseries_enbpi.py

```diff
@@ -0,0 +1,291 @@
+"""
+==================================================================
+Estimating prediction intervals of time series forecast with EnbPI
+==================================================================
+This example uses
+:class:`mapie.time_series_regression.MapieTimeSeriesRegressor` to estimate
+prediction intervals associated with time series forecast. It follows [6] and
+an alternative expermimental implemetation inspired from [2]
+
+We use here the Victoria electricity demand dataset used in the book
+"Forecasting: Principles and Practice" by R. J. Hyndman and G. Athanasopoulos.
+The electricity demand features daily and weekly seasonalities and is impacted
+by the temperature, considered here as a exogeneous variable.
+
+A Random Forest model is aloready fitted on data. The hyper-parameters are
+optimized with a :class:`sklearn.model_selection.RandomizedSearchCV` using a
+sequential :class:`sklearn.model_selection.TimeSeriesSplit` cross validation,
+in which the training set is prior to the validation set.
+The best model is then feeded into
+:class:`mapie.time_series_regression.MapieTimeSeriesRegressor` to estimate the
+associated prediction intervals. We compare four approaches: with or without
+``partial_fit`` called at every step, and following [6] or a approach inspired
+from [2]. It appears that the approach inspired from [2] and ``partial_fit``
+offer higher coverage, but with higher width of PIs and are much slower.
```

Comment:
I would stop the sentence after "of PIs" and then add a sentence starting with "In addition, the approach ..."

Reply:
I changed the documentation because method Plus from [2] is no longer relevant